added maya standalone node

wyoder opened the pull request:
`+`  We Added a node to `sub_graphs.nxt` called `/_maya_standalone_graph` to be able to kick off graphs in using a subprocess. This allows us to run graphs in parallel.

MichaelAldrich committed:
Merge pull request #94 from nxt-dev/dev

Release api-v0.13.0

ImLucasBrown committed:
Merge pull request #97 from nxt-dev/type_detection_patch

Release api-v0.13.1

ImLucasBrown committed:
Merge pull request #101 from nxt-dev/dev

Release api-v0.14.0

ImLucasBrown committed:
Merge pull request #105 from nxt-dev/dev

Release api-v0.15.0

ImLucasBrown committed:
* Bug fix: Avoid double comp when renaming a node.

MichaelAldrich committed:
Merge pull request #108 from nxt-dev/bug_double_comp_on_rename

Fix double comp when renaming

ImLucasBrown committed:
Merge pull request #109 from nxt-dev/dev

Release api-v0.15.1

ImLucasBrown committed:
Merge pull request #110 from nxt-dev/dev

Release api-v0.15.2

ImLucasBrown committed:
Merge pull request #111 from nxt-dev/dev

Release api-v0.15.3

wyoder committed:
added maya standalone node

wyoder committed:
updated arguments to match argument referenced names

ImLucasBrown commented on nxt/builtin/sub_graphs.nxt:
```diff
@@ -4,13 +4,99 @@
+                171.75387183252514
+            ]
+        }
+    },
```
We like to keep the builtin graphs free of meta data :)

wyoder replied:
This should be removed now.

ImLucasBrown reviewed (changes requested):
Small note, otherwise lgtm.

wyoder committed:
updated to pass json or string to parameters

Added some code to ensure json is being being used parameters if it's a dict type to convert it to a string. If it's already a string, we pass it as is and let the CLI deal with the rest

wyoder committed:
removed meta data from the graph
